test/compile/mark/mixins.test.ts

```diff
@@ -321,7 +321,7 @@ describe('compile/mark/mixins', () => {
       const model = parseUnitModelWithScaleAndLayoutSize({
         mark: {type: 'point'},
         encoding: {
-          color: {field: 'Foobar', type: 'nominal', legend: {title: 'baz', format: 's'}}
+          color: {field: 'Foobar', type: 'nominal', legend: {title: 'baz'}}
```

Comment:
Why does this have to be removed?

Reply:
Because otherwise we will format the field Foobar and the test will change. I have covered this case in the new test. I'm happy either way but this seemed to make more sense.